admin-src/utils/routing/getStateFromLocation.js

```diff
@@ -0,0 +1 @@
+export const getStateFromLocation = (location) => location.pathname.split('/')[1] || 'new'
```

Comment:
What is this function for? React router does this automatically. Don't try to parse URLs manually.